t/test-genome-DBs/homo_sapiens/core/table.sql

```diff
@@ -486,11 +486,11 @@ CREATE TABLE `meta` (
   `meta_id` int(11) NOT NULL AUTO_INCREMENT,
   `species_id` int(10) unsigned DEFAULT '1',
   `meta_key` varchar(40) NOT NULL,
```

Comment:
@nwillhoft  - the `meta_key` looks like always `VARCHAR(40)` whereas it should be `VARCHAR(64)

Reply:
I believe all the `meta_key` values have now been updated